addons/web_tour/static/src/js/tip.js

```diff
@@ -55,8 +55,8 @@ var Tip = Widget.extend({
     /**
      * @param {jQuery} $anchor the node on which the tip should be placed
      */
-    attach_to: async function ($anchor) {
-        this._setupAnchor($anchor);
+    attach_to: async function ($anchor, $altAnchor) {
```

Comment:
It's a shame that there isn't somewhere a complete documentation about how to write a tour, with a description of all keys available to define each step, where you could add the new alt_trigger key. However, you could at least adapt the docstring of this function accordingly (and maybe add a docstring for `update` as well).

Reply:
Sure!